packages/e2e/tsconfig.json

```diff
@@ -10,7 +10,9 @@
             "node",
             "cypress-file-upload"
         ],
-        "strictNullChecks": true /* Enable strict null checks. */
+        "strictNullChecks": true, /* Enable strict null checks. */
+        "esModuleInterop": true,
+        "resolveJsonModule": true,
```

Comment:
This is so I can do `import bigqueryCredentials from '../fixtures/credentials.json';` on warehouses.ts